core/src/main/java/org/elasticsearch/script/ScriptSettings.java

```diff
@@ -32,7 +32,8 @@
 
 public class ScriptSettings {
 
-    public static final String LEGACY_DEFAULT_LANG = "groovy";
+    static final String LEGACY_DEFAULT_LANG = "groovy";
```

Comment:
can you please put a big comment on this that this language is ONLY used for scripts stored in the percolator index etc. not for scripts coming in via the API